### PR TITLE
und-882 page hero feedbacks

### PR DESCRIPTION
**Pull request by Ashish-srij**
page hero feedbacks

### CODE REVIEW

File: stories/Molecules/Blocks/CountryCard/countrycard.scss [hunk 1]
```diff
@@ -16,7 +16,7 @@
 .country-card {
   @include devicebreak(medium) {
     &.glide__slide{
-      height: 100%;
+      //height: 100%;
```
Comment (shishirphadke) [inline stories/Molecules/Blocks/CountryCard/countrycard.scss:19]: @Ashish-srij Remove commented code.

File: stories/Molecules/Blocks/CountryCard/countrycard.scss [hunk 2]
```diff
@@ -43,7 +39,7 @@
       .country {
         &-card {
           &__caption {
-            transform: translateY(-100%);
+            transform: translateY(calc(-100% + 2.75rem));
```
Comment (shishirphadke) [inline stories/Molecules/Blocks/CountryCard/countrycard.scss:42]: @Ashish-srij On what condition is the value `2.75rem` based on? Can we use a global variable here?

Comment (Ashish-srij) [reply stories/Molecules/Blocks/CountryCard/countrycard.scss:42]: @shishirphadke we need to content move on hover and stay at bottom dynmicly. so we r using the calc here. and we don't have a variable for 2.75rem. I have tried to make a variable but variable is not working inside the calc().

File: stories/Molecules/Blocks/CountryCard/countrycard.scss (hunk 1 above)
Comment (Ashish-srij) [reply stories/Molecules/Blocks/CountryCard/countrycard.scss]: @shishirphadke done.

File: stories/Molecules/Blocks/CountryCard/countrycard.scss (hunk 2 above)
Comment (shishirphadke) [reply stories/Molecules/Blocks/CountryCard/countrycard.scss:41]: @Ashish-srij We can use it like this:
```
transform: translateY(calc(-100% + #{$spacing-44}));
```

**Pull request merged**